tools/migrations/24-09-24--adding-language-to-search.sql

```diff
@@ -0,0 +1,30 @@
+ALTER TABLE
+    `zeeguu_test`.`search`
+ADD
+    COLUMN `language_id` INT NULL
+AFTER
+    `id`,
+    CHANGE COLUMN `keywords` `keywords` VARCHAR(100) NULL DEFAULT NULL,
+ADD
+    INDEX `search_ibfk_1_idx` (`language_id` ASC) VISIBLE;
```

Comment:
i'm always getting syntax errors for the `VISIBLE`. i think we agreed that your DB management too likes it, but the mysql on the server does not like it. or is it a matter of mysql version? should we upgrade on the server?

Reply:
I am just using MySQLs workbench so that's just the default - which version of mysql is in the server?